packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -489,7 +534,30 @@ class SliverChildListDelegate extends SliverChildDelegate {
        assert(addAutomaticKeepAlives != null),
        assert(addRepaintBoundaries != null),
        assert(addSemanticIndexes != null),
-       assert(semanticIndexCallback != null);
+       assert(semanticIndexCallback != null),
+       _keyToIndex = <Key, int>{null: 0};
+
+  /// Creates a constant version of delegate that supplies children for slivers using the given
+  /// list.
+  ///
+  /// This constructor does not support children reordering.
+  ///
+  /// The [children], [addAutomaticKeepAlives], [addRepaintBoundaries],
+  /// [addSemanticIndexes], and [semanticIndexCallback] arguments must not be
+  /// null.
+  const SliverChildListDelegate.constant(
```

Comment:
rather than `constant`, how about `fixed`?

Talks about when you would use this, rather than what it is.

Reply:
fixed